src/strace.c

```diff
@@ -2460,6 +2470,18 @@ init(int argc, char *argv[])
 					  "build of strace");
 #endif
 			break;
+		case GETOPT_STACKTRACE_FRAMES_LIMIT:
+#ifdef ENABLE_STACKTRACE
+            i = string_to_uint(optarg);
```

Comment:
A minor cosmetic issue. `i = ...` should align to `case GETOPT_STRACKTRACE_FRAMES_LIMIT...`.